src/clients/base/base.ts

```diff
@@ -50,7 +57,9 @@ export class Base<OptionsType extends BaseOptions> extends EventEmitter {
 
   constructor (options: OptionsType) {
     super()
+    this[kClientType] = 'base'
     this[kInstance] = currentInstance++
+    this[kTracingChannel] = tracingChannel(`${clientsChannelName}:${this[kInstance]}`)
```

Comment:
That's not how this should work. The name should be fixed. I think you can drop the instanceId thing completely.